runtimes/terraform/tf/fargate.go

```diff
@@ -0,0 +1,80 @@
+package tf
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+import "github.com/falcosecurity/kilt/runtimes/cloudformation/cfnpatcher"
+
+type cfProperties struct {
+	RequiresCompatibilities []string                 `json:"RequiresCompatibilities"`
+	ContainerDefinitions    []map[string]interface{} `json:"ContainerDefinitions"`
+}
+
+type cfResource struct {
+	ResourceType string       `json:"Type"`
+	Properties   cfProperties `json:"Properties"`
+}
+
+type cfStack struct {
+	Resources map[string]cfResource `json:"Resources"`
+}
+
+func patchFargateTaskDefinition(ctx context.Context, containerDefinitions *types.String, kiltConfig *cfnpatcher.Configuration) (patched types.String, err error) {
```

Comment:
I don't like having named outputs. See [this recommendation](https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values).

Reply:
I don't like it either. I couldn't find any different way to return a value when cfnpatcher.Patch panics.